src/gui/accountsettings.cpp

```diff
@@ -719,9 +719,9 @@ void AccountSettings::slotOpenOC()
         QDesktopServices::openUrl(_OCUrl);
 }
 
-void AccountSettings::slotUpdateQuota(qint64 total, qint64 used)
+void AccountSettings::slotUpdateQuota(int quota, qint64 total, qint64 used)
```

Comment:
Why do we keep total? It's unused now if I'm not mistaken (should get a warning about that I guess).

Reply:
It is being used. But I also thought before that quota and total were giving different information and now they are the same?

Reply:
Well... before the `total` parameter on that function would receive the total value of the quota. That's why I would have expected `total` to be unused now with everything switched over to `quota`. But apparently this is not the case... so I guess the math is wrong somewhere now, we're showing the percentage of the total storage used and not the percentage of the quota used? That would be confusing I guess.